doc/install/python.rst

```diff
@@ -136,7 +136,7 @@ below demonstrate this for both pip and ``setup.py``.
     :ref:`Python configuration <install-python>` section of the
     :ref:`installation guide <in_build_install>`.
     CMake is recommended for developers, integration with package managers such as
-    Spack and EasyBuild, and users who require fine grained control over compilation
+    Spack and EasyBuild, and expert users who require fine grained control over compilation
```

Comment:
I'd like to be explicit: let's mention users who expect to use Arbor's C++ API.

Reply:
This is the installation guide for the Python wrapper, not the C++ installation guide.
So, this is targeting users who want fine-grained control over compilation options for their Python installation.

Reply:
Yes, but this like is telling them to look to the CMake build is they're 'expert users'. It's clearer if we say 'C++ users'.

Reply:
Oops, sorry for being a bit slow to reply. My point is that it isn't specifically for C++ users: it is for all users who want fine grained control

It is for users of Python, C++ and applications that use Arbor, that want fine grained control over the installation.

It is also for HPC users and admins who use spack, etc.

Reply:
Then let's leave the word out altogether: 'users who require fine grained control' is clear enough.

Reply:
Done!